bank_account_project.py

```diff
@@ -1,169 +0,0 @@
-'''
-***All credit for the data sourced for this project goes to exchangerates.org.uk***
-
-This program simulates a transaction with an ATM machine. If the user decides to 
-withdraw money from their bank account, they are given the option to enter the 
-withdrawal amount in a foreign currency. The program will grab the most updated 
-currency exchange figures and return the amount remaining in the bank account 
-in GBP. 
-'''
-
-#This function scrapes conversion rates between GBP(£) and some of the most 
-#common currencies in the world. The function takes one argument, gbp, that 
-#must be either an integar or a float. 
-def currency_func(gbp):
-
-    import requests
-    from bs4 import BeautifulSoup
-    
-    URL = 'https://www.exchangerates.org.uk/'
-    page = requests.get(URL,'html.parser')
-    soup = BeautifulSoup(page.content, 'html.parser')
-    
-    #grabbing the desired div and table for which the desired data is housed
-    table = soup.find('table')
-    
-    #grabbing all of the desired table data 
-    eur_exch = float(table.find_all('td')[3].text)
-    usd_exch = float(table.find_all('td')[8].text)
-    nzd_exch = float(table.find_all('td')[13].text)
-    aud_exch = float(table.find_all('td')[18].text)
-    cad_exch = float(table.find_all('td')[23].text)
-    jpy_exch = float(table.find_all('td')[28].text)
-    zar_exch = float(table.find_all('td')[33].text)
-    aed_exch = float(table.find_all('td')[38].text)
-    inr_exch = float(table.find_all('td')[43].text)
-    try_exch = float(table.find_all('td')[48].text)
-    chf_exch = float(table.find_all('td')[53].text)
```

Comment:
Think about using a for loop over a `dict` to avoid repeating similar lines over and over again. Your dict could have a `currency` as a key, and its index as a value (3, 8, 13, 18, etc). Here is what I am thiking:

```
valid_currencies_table_indices = {
  "EUR": 3,
  "USD": 8,
  "NZD": 13. ....
}
```